src/app/modules/consultations/consultation-profile/profile-card/profile-card.component.ts

```diff
@@ -113,12 +112,28 @@ export class ProfileCardComponent implements OnInit, OnChanges {
     }
   }
 
-  convertDateFormat(date) {
+  convertDateFormat(date: string): string {
     if (date) {
       return moment(date).format('Do MMM YY');
     }
+    return '';
+  }
+
+  convertToFormattedDate(date: string): string {
```

Comment:
The name is very similar to `convertDateFormat`. Rename `convertToFormattedDate` to make it clear for which purpose this date is used

Reply:
Same here

Reply:
@gpalsingh  - We are using this for different style of formatting